Userland/Libraries/LibGUI/EditingEngine.cpp

```diff
@@ -173,6 +173,8 @@ void EditingEngine::move_one_left()
         int new_column = m_editor->lines()[new_line].length();
         m_editor->set_cursor(new_line, new_column);
     }
+    if (m_editor->is_relative_line_number())
```

Comment:
Isn't it possible to put the code in `set_cursor`?

It will avoid quite the repetition

Reply:
I've already moved them into `void TextEditor::set_cursor(TextPosition const& a_position)`.